`make check` failure

```
cc1plus: warning: ./external/catch2/single_include/catch2/: No such file or directory [-Wmissing-include-dirs]
dynamical_decomposition.test.cc:24:10: fatal error: catch.hpp: No such file or directory
   24 | #include "catch.hpp"
      |          ^~~~~~~~~~~
compilation terminated.
```

Something has changed (probably my fault). check runs but fails
```
FAIL: interval_exchange_transformation.py
```
test-suite.log
```
=============================================
   pyintervalxt 0.0.1: test/test-suite.log
=============================================

# TOTAL: 1
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: interval_exchange_transformation.py
=========================================

============================= test session starts ==============================
platform linux -- Python 3.7.3, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /home/vincent/programming/flatsurf/intervalxt/pyintervalxt/test
collected 2 items

interval_exchange_transformation.py FF                                   [100%]

=================================== FAILURES ===================================
_____________________ test_IntervalExchangeTransformation ______________________

    def test_IntervalExchangeTransformation():
>       iet = intervalxt.IET((18, 3), (1, 0))

interval_exchange_transformation.py:30: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

lengths = (18, 3), permutation = (1, 0)

    def IET(lengths, permutation):
        from cppyy.gbl import std
>       return intervalxt.makeIET(intervalxt.sample.Lengths[int](lengths), permutation)
E       AttributeError: <namespace cppyy.gbl.intervalxt at 0x56346f365bc8> has no attribute 'makeIET'. Full details:
E         type object 'intervalxt' has no attribute 'makeIET'
E         'intervalxt::makeIET' is not a known C++ class
E         'makeIET' is not a known C++ template
E         'makeIET' is not a known C++ enum

../src/pyintervalxt/cppyy_intervalxt.py:60: AttributeError
_________________________________ test_reduce __________________________________

    def test_reduce():
>       iet = intervalxt.IET((18, 3), (1, 0))

interval_exchange_transformation.py:36: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

lengths = (18, 3), permutation = (1, 0)

    def IET(lengths, permutation):
        from cppyy.gbl import std
>       return intervalxt.makeIET(intervalxt.sample.Lengths[int](lengths), permutation)
E       AttributeError: <namespace cppyy.gbl.intervalxt at 0x56346f365bc8> has no attribute 'makeIET'. Full details:
E         type object 'intervalxt' has no attribute 'makeIET'
E         'intervalxt::makeIET' is not a known C++ class
E         'makeIET' is not a known C++ template
E         'makeIET' is not a known C++ enum

../src/pyintervalxt/cppyy_intervalxt.py:60: AttributeError
============================== 2 failed in 0.09s ===============================
FAIL interval_exchange_transformation.py (exit status: 1)
```

--- a/pyintervalxt/src/pyintervalxt/cppyy_intervalxt.py
+++ b/pyintervalxt/src/pyintervalxt/cppyy_intervalxt.py
@@ -48,9 +48,8 @@ def pretty_print(proxy, name):
 
 cppyy.py.add_pythonization(pretty_print, "intervalxt")
 
-for path in os.environ.get('PYINTERVALXT_INCLUDE','').split(':'):
-    if path: cppyy.add_include_path(path)
-
+# Set EXTRA_CLING_ARGS="-I /usr/include" or wherever intervalxt/cppyy.hpp can
+# be resolved if the following line fails to find the header file.
 cppyy.include("intervalxt/cppyy.hpp")
 
 from cppyy.gbl import intervalxt

--- a/pyintervalxt/test/Makefile.am
+++ b/pyintervalxt/test/Makefile.am
@@ -11,4 +11,4 @@ BUILT_SOURCES = test-env.sh
 EXTRA_DIST += test-env.sh.in
 CLEANFILES = test-env.sh
 $(builddir)/test-env.sh: $(srcdir)/test-env.sh.in Makefile
-	sed -e 's,[@]srcdir[@],$(srcdir),g' -e 's,[@]builddir[@],$(builddir),g' -e 's,[@]pythondir[@],$(pythondir),g' < $< > $@
+	sed -e 's,[@]abs_srcdir[@],$(abs_srcdir),g' -e 's,[@]abs_builddir[@],$(abs_builddir),g' -e 's,[@]pythondir[@],$(pythondir),g' < $< > $@

--- a/pyintervalxt/test/test-env.sh.in
+++ b/pyintervalxt/test/test-env.sh.in
@@ -1,5 +1,7 @@
 #!/bin/sh
 
-export PYINTERVALXT_INCLUDE="@srcdir@/../../libintervalxt:@builddir@/../../libintervalxt/intervalxt"
-export LD_LIBRARY_PATH="@builddir@/../../libintervalxt/src/.libs/"
-export PYTHONPATH="@builddir@/../../pyintervalxt/src/:@pythondir@"
+# Resolve #include <intervalxt/*.hpp> relative to libintervalxt in the source tree and
+# resolve #include "intervalxt.hpp" relative to libintervalxt/intervalxt in the build tree.
+export EXTRA_CLING_ARGS="-I@abs_srcdir@/../../libintervalxt -I@abs_builddir@/../../libintervalxt/intervalxt $EXTRA_CLING_ARGS"
+export LD_LIBRARY_PATH="@abs_builddir@/../../libintervalxt/src/.libs/:$LD_LIBRARY_PATH"
+export PYTHONPATH="@abs_srcdir@/../src/:@pythondir@"

